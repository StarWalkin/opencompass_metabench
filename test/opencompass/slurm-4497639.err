  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]slurmstepd: error: *** JOB 4497639 ON gpua079 CANCELLED AT 2024-08-17T00:08:53 ***
