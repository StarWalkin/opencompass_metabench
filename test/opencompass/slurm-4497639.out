/u/zzhang27/.conda/envs/opencompass/bin/python3
08/16 23:14:17 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/16 23:14:17 - OpenCompass - INFO - Loading hf_gemma2_9b_it: configs/models/gemma/hf_gemma2_9b_it.py
08/16 23:14:17 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/16 23:14:17 - OpenCompass - INFO - Current exp folder: outputs/default/20240816_231417
08/16 23:14:17 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/16 23:14:18 - OpenCompass - INFO - Partitioned into 1 tasks.
